src/lib/structures/HelpPaginatedMessage.ts

```diff
@@ -0,0 +1,240 @@
+import { SkyraPaginatedMessage } from './SkyraPaginatedMessage';
+
+export class HelpPaginatedMessage extends SkyraPaginatedMessage {
+	private language: TFunction;
+
+	public constructor(language: TFunction, options: PaginatedMessageOptions = {}) {
+		super(options);
+		this.setIdle(minutes(10));
+		this.language = language;
+
+		this.actions = new Map<string, HelpPaginatedMessageAction>([
+			[
+				'@sapphire/paginated-messages.goToPage',
+				{
+					customId: '@sapphire/paginated-messages.goToPage',
+					type: Constants.MessageComponentTypes.SELECT_MENU,
+					selectMenuIndex: 'set-1',
+					run: ({ handler, interaction }) => interaction.isSelectMenu() && (handler.index = parseInt(interaction.values[0], 10))
+				}
+			],
+			[
+				'@sapphire/paginated-messages.goToPage-2',
+				{
+					customId: '@sapphire/paginated-messages.goToPage-2',
+					type: Constants.MessageComponentTypes.SELECT_MENU,
+					selectMenuIndex: 'set-2',
+					run: ({ handler, interaction }) => interaction.isSelectMenu() && (handler.index = parseInt(interaction.values[0], 10))
+				}
+			],
+			[
+				'@sapphire/paginated-messages.firstPage',
+				{
+					customId: '@sapphire/paginated-messages.firstPage',
+					style: 'PRIMARY',
+					emoji: '⏪',
+					type: Constants.MessageComponentTypes.BUTTON,
+					run: ({ handler }) => (handler.index = 0)
+				}
+			],
+			[
+				'@sapphire/paginated-messages.previousPage',
+				{
+					customId: '@sapphire/paginated-messages.previousPage',
+					style: 'PRIMARY',
+					emoji: '◀️',
+					type: Constants.MessageComponentTypes.BUTTON,
+					run: ({ handler }) => {
+						if (handler.index === 0) {
+							handler.index = handler.pages.length - 1;
+						} else {
+							--handler.index;
+						}
+					}
+				}
+			],
+			[
+				'@sapphire/paginated-messages.nextPage',
+				{
+					customId: '@sapphire/paginated-messages.nextPage',
+					style: 'PRIMARY',
+					emoji: '▶️',
+					type: Constants.MessageComponentTypes.BUTTON,
+					run: ({ handler }) => {
+						if (handler.index === handler.pages.length - 1) {
+							handler.index = 0;
+						} else {
+							++handler.index;
+						}
+					}
+				}
+			],
+			[
+				'@sapphire/paginated-messages.goToLastPage',
+				{
+					customId: '@sapphire/paginated-messages.goToLastPage',
+					style: 'PRIMARY',
+					emoji: '⏩',
+					type: Constants.MessageComponentTypes.BUTTON,
+					run: ({ handler }) => (handler.index = handler.pages.length - 1)
+				}
+			],
+			[
+				'@sapphire/paginated-messages.stop',
+				{
+					customId: '@sapphire/paginated-messages.stop',
+					style: 'DANGER',
+					emoji: '⏹️',
+					type: Constants.MessageComponentTypes.BUTTON,
+					run: async ({ collector, response }) => {
+						collector.stop();
+						await response.edit({ components: [] });
+					}
+				}
+			]
+		]);
+	}
+
+	public override addPage(page: PaginatedMessagePage): this {
+		this.pages.push(page);
+
+		return this;
+	}
```

Comment:
Useless?

Reply:
No, because the default `addPage` has the emitWarning for exceeding 25 pages.